components/02-molecules/article&blog-post-elements/wall-of-consent/wall-of-consent.js

```diff
@@ -8,12 +8,14 @@ export default class WallOfConsent extends HTMLElement {
     }
 
     this.checkbox.addEventListener('change', this.toggle.bind(this))
-    document.body.addEventListener('consent-to-embeds', this.revealContent.bind(this))
-    document.body.addEventListener('disconsent-to-embeds', this.removeContent.bind(this))
+    ;['revealContent', 'removeContent'].forEach(prop => {
```

Comment:
Is the semicolon needed in line 11?

Reply:
Only because y'all insist on not using semicolons properly... 🙂 

Otherwise the interpreter will try to join this with the previous line, thinking this is a property accessor (bracket notation).

(For details, look for "ASI in JavaScript" (2016) on our internal mailing list.